frontend/src/DialectReportView.tsx

```diff
@@ -37,7 +37,9 @@ export const DialectReportView = ({
   return (
     <main className="container-lg">
       <div className="col col-lg-8 mx-auto">
-        <BowtieInfoSection />
+        {topPageInfoComponent &&
```

Comment:
Wouldn't it work if you just tried to render this info component? Without the additional checks? If it's null or undefined then it just doesn't get rendered I think?